Backend/dev/migrations-read-only/rider-dashboard/API_Management_InsertPartnerOrgStation.sql

```diff
@@ -0,0 +1,8 @@
+-- {"api":"PostInsertPartnerOrgStationInsertPartnerOrgStation","migration":"userActionType","param":"ApiAuth APP_BACKEND_MANAGEMENT MIGRATION INSERT_PARTNER_ORG_STATION","schema":"atlas_bap_dashboard"}
+INSERT INTO atlas_bap_dashboard.access_matrix (id, role_id, api_entity, user_access_type, user_action_type) ( SELECT atlas_bap_dashboard.uuid_generate_v4(), T1.role_id, 'DSL', 'USER_FULL_ACCESS', 'RIDER_MANAGEMENT/INSERT_PARTNER_ORG_STATION/POST_INSERT_PARTNER_ORG_STATION_INSERT_PARTNER_ORG_STATION' FROM atlas_bap_dashboard.access_matrix AS T1 WHERE T1.user_access_type = 'USER_FULL_ACCESS' AND T1.api_entity = 'MIGRATION' AND T1.user_action_type = 'INSERT_PARTNER_ORG_STATION' ) ON CONFLICT DO NOTHING;
```

Comment:
revert the sql file change and regenerate it, as it has couple of migrations, but it should be only one.